docs/contributors/getting-started.md

```diff
@@ -31,54 +30,30 @@ To install Docker Compose, [follow their instructions here](https://docs.docker.
 Once Docker is installed and running, run this script to install WordPress, and build your local environment:
 
 ```
-npm run env install
+npx wp-env start
 ```
-
-#### Alternative Method: Using an Existing Local WordPress Install
-WordPress will be installed in the `wordpress` directory, if you need to access WordPress core files directly, you can find them there.
-
-If you already have WordPress checked out in a different directory, you can use that installation, instead, by running these commands:
-
-```
-export WP_DEVELOP_DIR=/path/to/wordpress-develop
-npm run env connect
-```
-
-This will use WordPress' own local environment, and mount your Gutenberg directory as a volume there.
-
-In Windows, you can set the `WP_DEVELOP_DIR` environment variable using the appropriate method for your shell:
-
-    CMD: set WP_DEVELOP_DIR=/path/to/wordpress-develop
-    PowerShell: $env:WP_DEVELOP_DIR = "/path/to/wordpress-develop"
 	
 ### Step 2: Accessing and Configuring the Local WordPress Install
 #### Accessing the Local WordPress Install
 
-Whether you decided to use Docker or an existing local WordPress install, the WordPress installation should now be available at `http://localhost:8889` (**Username**: `admin`, **Password**: `password`).
-If this port is in use, you can override it using the `LOCAL_PORT` environment variable. For example running the below command on your computer will change the URL to
-`http://localhost:7777` .
-
-Linux/macOS: `export LOCAL_PORT=7777`
-Windows using Command Prompt: `setx LOCAL_PORT "7777"`
-Windows using PowerShell: `$env:LOCAL_PORT = "7777"`
-
-If you're running [e2e tests](/docs/contributors/testing-overview.md#end-to-end-testing), this change will be used correctly.
+The WordPress installation should now be available at `http://localhost:8888` (**Username**: `admin`, **Password**: `password`).
+If this port is in use, you can override it using the `WP_ENV_PORT` environment variable. For more information, consult the `wp-env` [README](https://github.com/WordPress/gutenberg/blob/master/packages/env/README.md).
 
-To shut down this local WordPress instance run `npm run env stop`. To start it back up again, run `npm run env start`.
+To shut down this local WordPress instance run `npx wp-env stop`. To start it back up again, run `npx wp-env start` again.
 
 #### Toggling Debug Systems
 
 WordPress comes with specific [debug systems](https://wordpress.org/support/article/debugging-in-wordpress/) designed to simplify the process as well as standardize code across core, plugins and themes. It is possible to use environment variables (`LOCAL_WP_DEBUG` and `LOCAL_SCRIPT_DEBUG`) to update a site's configuration constants located in `wp-config.php` file. Both flags can be disabled at any time by running the following command:
 
 Example on Linux/MacOS:
 ```
-LOCAL_SCRIPT_DEBUG=false LOCAL_WP_DEBUG=false npm run env install
+LOCAL_SCRIPT_DEBUG=false LOCAL_WP_DEBUG=false npx wp-env start
```

Comment:
Does this work with `wp-env`?

Reply:
No, you have to navigate to the installation and edit the `wp-config.php` file for now.

@noisysocks We could add support to the `wp-env` config file though, right?

Reply:
Yes it's on my list of follow-up work once #20002 is merged! 🙂

Reply:
Updated for now: 65f5ad065